doc/source/basics.rst

```diff
@@ -64,37 +64,37 @@ NumPy's type system to add support for custom arrays
 (see :ref:`basics.dtypes`).
 
 To get the actual data inside a :class:`Index` or :class:`Series`, use
-the **array** property
+the ``.array`` property
 
 .. ipython:: python
 
    s.array
    s.index.array
 
 :attr:`~Series.array` will always be an :class:`~pandas.api.extensions.ExtensionArray`.
-The exact details of what an ``ExtensionArray`` is and why pandas uses them is a bit
+The exact details of what an :class:`~pandas.api.extensions.ExtensionArray` is and why pandas uses them is a bit
 beyond the scope of this introduction. See :ref:`basics.dtypes` for more.
 
 If you know you need a NumPy array, use :meth:`~Series.to_numpy`
-or :meth:`numpy.asarray`.
+or :meth:`np.asarray`.
```

Comment:
did you check if this renders correctly? I think we need to use `numpy` and not `np` here.

Reply:
doesn’t render now with numpy

Reply:
I see using `numpy` creating the links, for example: https://pandas-docs.github.io/pandas-docs-travis/basics.html#searchsorted

Reply:
That's weird, not sure why in this case it's not working. But if `np` works in this case, that's good enough for now (in the future would be good to understand what's going on, and make it work in all the cases for one of them).

Reply:
need numpy.ndarray.asarray i think